functions/helpers.ts

```diff
@@ -6,7 +6,9 @@ import { getGiftCardById } from "./post-order";
 import { fallbackIntlMastercard, fallbackIntlVisa, masterCardIntlSkus, visaIntlSkus } from "./reloadly-lists";
 import { AccessToken, ReloadlyFailureResponse } from "./types";
 
-export const allowedChainIds = [1, 5, 100, 31337];
+export const permitAllowedChainIds = [1, 5, 100, 31337];
+
+export const ubiquityDollarAllowedChainIds = [1, 31337];
```

Comment:
We bridged it to gnosis chain but it has a different address 

https://gnosisscan.io/address/0xc6ed4f520f6a4e4dc27273509239b7f8a68d2068